### PR TITLE
[WFLY-18904]: Update ANTLR to 4.13.0 for Hibernate 6.4

### PR DESCRIPTION
**Pull request by nekop**
https://issues.redhat.com/browse/WFLY-18904

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -361,7 +361,7 @@
             Properties for dependencies that solely relate to maven plugins go in the plugin section above.
             Properties for dependencies that solely relate to tests go in the test code section above.
          -->
-        <version.antlr>4.10</version.antlr>
+        <version.antlr>4.14.0</version.antlr>
```
Comment (scottmarlow) [inline pom.xml:364]: Thanks @nekop  for figuring out that we need to update ANTLR!

Please change to use Antlr version 4.13.0 as is used in https://github.com/hibernate/hibernate-orm/blob/6.4/settings.gradle#L67 and also is what the WFLY-18904 issue states as well.

Comment (nekop) [reply pom.xml]: Woops, terrible typo. Fixed.

Comment (scottmarlow) [reply pom.xml]: Thank you for the change!

**Review by scottmarlow — APPROVED** [review]

**Review by scottmarlow — APPROVED** [review]

**Comment by scottmarlow** [comment]
@bstansberry I think this is ready for merging for WildFly 31.0.0.Final, thanks!

**Comment by bstansberry** [comment]
Thanks @nekop and @scottmarlow!

**Pull request merged**